test/integration/console_sock_test.lua

```diff
@@ -20,11 +21,29 @@ g.before_all = function()
         http_port = 8080,
         cluster_cookie = 'super-cluster-cookie',
         env = {
-            TARANTOOL_CONSOLE_SOCK = fio.pathjoin(tmpdir, '/foo.sock')
+            TARANTOOL_CONSOLE_SOCK = fio.pathjoin(tmpdir, '/foo.sock'),
+            NOTIFY_SOCKET = fio.pathjoin(tmpdir, '/notify.sock')
         },
     })
+    -- build notify socket
+    local notify_socket = assert(socket('AF_UNIX', 'SOCK_DGRAM', 0), 'Can not create socket')
+    if fio.stat(server.env.NOTIFY_SOCKET) then
+        assert(fio.unlink(server.env.NOTIFY_SOCKET))
+    end
+    assert(notify_socket:bind('unix/', server.env.NOTIFY_SOCKET), notify_socket:error())
+    fio.chmod(server.env.NOTIFY_SOCKET, tonumber('0666', 8))
     server:start()
-    t.helpers.retrying({}, function() server:graphql({query = '{}'}) end)
+    -- wait notify
+    while true do
```

Comment:
use `retrying()` as before instead of `while true`. Imagine the notification stops working for some reason one day. So the test will hang until gitlab kills it. You should rely on `retrying()` instead.